### DISCUSSION TITLE
NPC 복제시 나타나는 에러

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#478] [5. Q&A]
Not allowed to access vertices on mesh 'HANS_rig_re(Clone)' (isReadable is false; Read/Write must be enabled in import settings)

실행 : 제페토 id를 통한 NPC 복제
- 실행 자체는 되는데, 해당 에러가 왜 나타나는 지 아시는 분 있나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 에러 메세지만 보았을때 'HANS_rig_re(Clone)'  이라는 메시를 검색하신 후, 인스펙터에서 "read\write enabled" 옵션을 체크해보시면 도움이 될 것 같습니다.
